source/Calamari.Tests/KubernetesFixtures/Helm3UpgradeFixture.cs

```diff
@@ -111,6 +111,66 @@ public async Task HelmVersionOlderThanMinimumVersion_DoesNotRunObjectStatus()
             }
         }
 
+        [Test]
+        [RequiresNonFreeBSDPlatform]
+        [RequiresNon32BitWindows]
+        [RequiresNonMac]
+        [Category(TestCategory.PlatformAgnostic)]
```

Comment:
What does it mean for a test to be platform agnostic but still require a 64-bit Windows environment?

Reply:
I have no idea 🤣